### PR TITLE
[4.0] Language labels uppercase

### PR DESCRIPTION
**Pull request by pe7er**
Codestyle fix for language labels in.xml files

### Summary of Changes
Some language labels in .xml files were written in lowercase.
This PR is to change them to UPPERCASE.

### CODE REVIEW

**Comment by grhcj** [comment]
I have tested this item :white_check_mark: successfully on 8e4c42e4713621e2b1ea8aa80ae95315de7e2bb6<br /><br />On code Review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24815">issues.joomla.org/tracker/joomla-cms/24815</a>.</sub>

File: components/com_users/tmpl/login/default.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metadata>
-	<layout title="com_user_login_view_default_title" option="com_user_login_view_default_option">
+	<layout title="COM_USER_LOGIN_VIEW_DEFAULT_TITLE" option="COM_USER_LOGIN_VIEW_DEFAULT_OPTION">
```
Comment (Quy) [inline components/com_users/tmpl/login/default.xml:3]: Change to `USERS`?

Comment (pe7er) [reply components/com_users/tmpl/login/default.xml:3]: Thanks, I did that in another PR: https://github.com/joomla/joomla-cms/pull/24805

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**